javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -14,19 +14,201 @@
 
 package build.bazel.tests.integration;
 
+import org.hamcrest.Description;
+import org.hamcrest.SelfDescribing;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+import static org.hamcrest.core.Is.is;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 import static com.google.common.truth.Truth.assertThat;
 import static org.junit.Assert.assertEquals;
 
-import org.junit.Test;
-
-/** {@link BazelBaseTestCase}Test */
+/**
+ * {@link BazelBaseTestCase}Test
+ */
+//suppress since same parameter value is ok for tests readability, tests should encapsulate and not hide
+@SuppressWarnings("SameParameterValue")
```

Comment:
What is that Warning? Eclipse says it doesn't know it.

Reply:
It's an intellij one.
It's about having a parameter with only one usage so it suggests to inline the parameter.